README.md

```diff
@@ -14,15 +14,15 @@ except for some cases concerning PSQL, wherein the query itself is translated.
 
 Simple query mode and extended query mode are supported, and any data type
 supported by Spanner is also supported. Items, tables and language not native to
-Spanner are not supported, unless otherwise specified.
+Spanner is not supported, unless otherwise specified.
 
 Though the majority of functionality inherent in most PostgreSQL clients
 (including PSQL and JDBC) are included out of the box, the following items are
 not supported:
-* Functions
-* COPY protocol
-* Prepared Statement DESCRIBE
-* SSL
+* Functions.
```

Comment:
Nit: bulleted lists of nouns don't often have '.'s

Reply:
idem